src/effects/ObjectDetection.cpp

```diff
@@ -133,29 +157,32 @@ bool ObjectDetection::LoadObjDetectdData(std::string inputFilePath){
     // Create tracker message
     pb_objdetect::ObjDetect objMessage;
 
-    {
-        // Read the existing tracker message.
-        fstream input(inputFilePath, ios::in | ios::binary);
-        if (!objMessage.ParseFromIstream(&input)) {
-            cerr << "Failed to parse protobuf message." << endl;
-            return false;
-        }
+    
+    // Read the existing tracker message.
+    fstream input(inputFilePath, ios::in | ios::binary);
+    if (!objMessage.ParseFromIstream(&input)) {
+        cerr << "Failed to parse protobuf message." << endl;
+        return false;
```

Comment:
Should this return an openshot::Exception instead? If for some reason, it fails here, I just want to make sure it bubbles up clearly/loudly.

Reply:
@jonoomph I guess my question would be, how common a failure is it?

IOW, is it truly an exceptional condition? Or are there cases where it's "normal" that we can't parse certain files, for reasons? (In which instance, I'd argue for _at most_ a debug log message, and even then only if we **also** log successful parse attempts.)

If it truly is an exceptional this-shouldn't-happen, then agreed: 
```cpp
// 😝
return false;
// 🤩
throw ProtobufParseError("Failed to parse protobuf message", inputFilePath);
```